ext/net/02_tls.js

```diff
@@ -92,6 +106,13 @@ async function startTls(
     alpnProtocols = undefined,
   } = {},
 ) {
+  if (certFile !== undefined) {
+    internals.warnOnDeprecatedApi(
+      "Deno.ListenTlsOptions.certFile",
```

Comment:
Ditto

Reply:
Actually, I removed this change. See #22032.